app/models/world_location.rb

```diff
@@ -52,28 +53,28 @@ def self.cache_fetch(key)
     end
   end
 
-  def initialize(data)
-    @data = data
-  end
-
-  def slug
-    self.details["slug"]
+  def initialize(location)
+    @location = location
   end
 
   def ==(other)
-    other.is_a?(self.class) && other.slug == slug
+    other.is_a?(self.class) && other.slug == self.slug
   end
 
-  def_delegators :@data, :title, :details
-  # def_delegators :details, :slug
+  def_delegators :@location, :title, :details, :slug
   alias_method :name, :title
 
   def organisations
-    # binding.pry
-    @organisations ||= WorldwideOrganisation.for_location(slug)
+    @organisations ||= WorldwideOrganisation.for_location(self.slug)
```

Comment:
Yeah, I have change slug to be an instance variable see https://github.com/alphagov/smart-answers/blob/3c8ff7a363617d959145b70ac58a7d934757b1ec/app/models/world_location.rb#L61